eng/common/scripts/get-codeowners.ps1

```diff
@@ -26,15 +25,19 @@ foreach ($contentLine in $codeOwnersContent) {
   }
 }
 
-$results = $ownedFolders[$TargetDirectory.ToLower()]
+$matchedKey = $ownedFolders.keys `
+  | ? { $target.StartsWith($_) } `
+  | Sort-Object -Property Length -Descending `
```

Comment:
Don't you want the shortest match? In which case you wouldn't want the descending option.

Reply:
Negative sir! I want the longest match! More exact the better!

Reply:
We talked offline. Updates complete.